src/Umbraco.Cms.Integrations.Crm.Dynamics/Helpers/EmbedCodeExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Umbraco.Cms.Integrations.Crm.Dynamics.Helpers
+{
+    public static class EmbedCodeExtensions
+    {
+        /// <summary>
+        /// Read <div></div> tags attributes like id, class, data- from input HTML using Regular Expressions.
+        /// </summary>
+        /// <param name="html"></param>
+        /// <param name="attributeKey"></param>
+        /// <returns></returns>
+        public static string ParseAttributeValue(this string html, string attributeKey)
+        {
+            string divPattern = @"<\s*{0}[^>]*>(.*?)<\s*/{0}\s*>";
```

Comment:
Just wondering if this could be treated as XML, and if so parsing the string into an `XDocument`?  If so, from there it may be easier to work with than with regular expressions.

Would suggest also creating a unit test for this method use to verify behaviour.